doccano_client/beta/controllers/comment.py

```diff
@@ -57,20 +62,27 @@ def all(self) -> Iterable[CommentController]:
             CommentController: The next comment controller.
         """
         response = self.client_session.get(self.comments_url)
-        verbose_raise_for_status(response)
-        comment_dicts = response.json()
-        comment_obj_fields = set(comment_field.name for comment_field in fields(Comment))
-
-        for comment_dict in comment_dicts:
-            # Sanitize comment_dict before converting to Comment
-            sanitized_comment_dict = {comment_key: comment_dict[comment_key] for comment_key in comment_obj_fields}
-
-            yield CommentController(
-                comment=Comment(**sanitized_comment_dict),
-                username=comment_dict["username"],
-                created_at=comment_dict["created_at"],
-                example=comment_dict["example"],
-                id=comment_dict["id"],
-                comments_url=self.comments_url,
-                client_session=self.client_session,
-            )
+
+        while True:
```

Comment:
Quick comment here would be helpful explaining why we have a while true loop, explaining that we don't get all comments back at once and need to call the next url sent back by django